app/controllers/requests_controller.rb

```diff
@@ -23,8 +23,9 @@ def edit
   # POST /requests
   def create
     @request = Request.new(request_params)
-
     if @request.save
+      Mailer.send_generic_email(false, @request.email, 'workshop.portal@gmail.com', I18n.t('requests.email.topic'),
```

Comment:
The problem is that the @gmail address was planed as just an internal email address for sending. So there was no intention to ever have to change it. 
But in this case it would be a good idea to make the reply_to address configurable by organizers/admins